src/HomeWork.java

```diff
@@ -0,0 +1,91 @@
+
+public class HomeWork {
+    public static int calculateAbsoluteDifference(int a, int b) {
+        return Math.abs(a - b);
+    }
+
+    public static void main(String[] args) {
+        int a = 5;
+        int b = 9;
+        int result = calculateAbsoluteDifference(a, b);
+
+        System.out.println("a = " + a);
+        System.out.println("a = " + b);
+        System.out.println("result = " + result);
+
+        a = -8;
+        b = 7;
+        result = calculateAbsoluteDifference(a, b);
+
+        System.out.println("a = " + a);
+        System.out.println("a = " + b);
+        System.out.println("result = " + result);
+
+    }
+}
+
+class FirstChar {
+    public static char getFirstCharacter(String inputString) {
+        if (inputString != null && !inputString.isEmpty()) {
+            char[] charArray = inputString.toCharArray();
+            return charArray[0];
+        } else {
+            throw new IllegalArgumentException(" ");
```

Comment:
This is really over complicated. It works but not really correct. Take a look at the second task that we did during the lesson. We did it specifically to help all of you. No need to turn everything to char array. No need to check if the string is null or empty, we haven't learned that yet and we assume that string always has a value.